### ISSUE TITLE
Only 2 empty folders created when exporting to file

### ISSUE THREAD

**Issue by b-per** [didierterrien/Power-BI-Sidetools#14]
Hi!

When trying to export my (big = 700MB) model to files, only 2 root folders get created, the M and the DAX ones, and then only 1 subfolder is created under M, QG_ Diagnostics. And none of those folders contains any file.

Would there be a way to know why it is failing?

Here is the log:
```
=== V1.2.1.0 ======================
[05-10-2021 18:10:24]  Folders initialised
[05-10-2021 18:10:25]  Power BI file name identified
[05-10-2021 18:10:26]  Connected
[05-10-2021 18:10:26]  Initialisation completed
[05-10-2021 18:10:26]  Updates checked
```

**Comment by didierterrien** [comment]
Hello,
Thanks for the feedback.
Please confirm your model is imported in the desktop.
Does it work with an other pbix using import mode ?

**Comment by b-per** [comment]
Yes, the model is imported, with a SQL DB as the source. 
I currently don't have the credentials to the SQL DB though, if it makes a difference.

And, yes also, the export to files works for other PBI files that I have.

**Comment by b-per** [comment]
After a bit of troubleshooting I found the issue. In "Transform Data", most of my queries are imported from a SQL DB but just a few sources are offline JSON files. After removing those queries from my list (which breaks my model), the PBI to files export works as expected, 

Unfortunately, I can't send the file without spending a fair bit of time sanitising the data as it contains confidential information.

**Comment by didierterrien** [comment]
I don't understand why some queries prevent PBI to files working. If you will have the opportunity to send a pbix in the future, I will check with pleasure. I close the issue for the moment.

**Comment by b-per** [comment]
Hi!

 I managed to create a minimal file replicating the issue that I raised a few days ago. What would be the best way to share the PBIX? I can't attach it to the issue itself unfortunately.

**Comment by b-per** [comment]
Hi Didier,

Here is the zipped file (still failing with 1.2.3)
[Example Failed Export.zip](https://github.com/didierterrien/Power-BI-Sidetools/files/6652958/Example.Failed.Export.zip)

**Comment by didierterrien** [comment]
Hi,
Thank you ! I will check it quickly

**Comment by didierterrien** [comment]
Issue is solved in version 1.2.4. I will publish it soon.
Much much easier with a sample file ! Thank you for your contribution.

**Comment by didierterrien** [comment]
I just published version 1.2.4 solving this issue
Thanks for the feedback !
